lib/wrappers/python/README.md

```diff
@@ -9,6 +9,37 @@
 5. Move compiled library located in `target/release` directory to same directory as `contourwall.py`
 6. Run: `python3 demo.py`
 
+## Example Usage
+Create a new file and use the import 'from contourwall import ContourWall, hsv_to_rgb', to be able to use the python wrapper.
+
+To utilize this library, ensure you have the necessary serial port permissions and ESP32 firmware flashed with the ContourWall protocol support.
+
+``` Python
+from contourwall import ContourWall, hsv_to_rgb
+def fade_colors():
+    # Slowly fade over all HSV colors
+    for i in range(0, 360):
+        cw.pixels[:] =  hsv_to_rgb(i, 100, 100)
+        cw.show()
+    cw.fill_solid(0, 0, 0)
+    cw.show(sleep_ms=10)
+```
+
+## Functions in the python wrapper
+|Type|Function|Description|
```

Comment:
This is fine, however, your table includes more then functions, it also includes classes. Therefore rename it to something like: "Functions and classes in Python wrapper"

Same for the labels of the table